core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -1136,4 +1138,18 @@ public FeaturesConfig setIncrementalHashArrayLoadFactorEnabled(boolean increment
         this.incrementalHashArrayLoadFactorEnabled = incrementalHashArrayLoadFactorEnabled;
         return this;
     }
+
+    @NotNull
+    public boolean isHideInaccesibleColumns()
+    {
+        return hideInaccesibleColumns;
+    }
+
+    @Config("hide-inaccessible-columns")
+    @ConfigDescription("When enabled non accessible columns will be filtered in SELECT * statement")
```

Comment:
I am not a native speaker, but when I read it now I under impression that the below would sound better. WDYT?

> When enabled non accessible columns will be filtered from SELECT * statement

Reply:
ok